spec/frodo/profile_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+describe Frodo::Profile do
+  let(:location) do
+    OpenStruct.new(id: 'asdf')
+  end
+
+  let(:token) { 'Foobarbaz' }
+
+  let(:headers) do
+    { Authorization: "Bearer #{token}" }
+  end
+
+  before do
+    ENV['GANDALF_URL'] = 'https://www.gandalf.com'
+    stub_request(:get, ENV['GANDALF_URL'] + '/location/asdf/profile')
+      .with(headers: headers)
+      .to_return(status: status, body: body, headers: {})
+  end
+
+  context '#profile' do
```

Comment:
This still is not a method name 😛  but i'll pass it.

I'll just fix it later if I ever do something in this repo.